chain/chain_rpc.js

```diff
@@ -53,7 +53,16 @@ module.exports = ({cmd, network, params}, cbk) => {
       called = true;
 
       if (!response) {
-        return cbk([503, 'ExpectedNonEmptyChainResponse']);
+        try {
```

Comment:
I don't think there needs to be try here - I think I added the above one in debugging and it isn't needed